server/admin_management/api/app/services/access_control_privilege_service.py

```diff
@@ -157,3 +176,45 @@ def grant_privilege(
             )
 
         return access_control_privilege_return
+
+
+class ForestClientValidator:
```

Comment:
Wondering maybe you like to have this new validator into a folder "/services/validators" so it does not need to be located in this service? Later it may come in handy for business rule validations for other services.